### ISSUE TITLE
What's the format?

### ISSUE THREAD

**Issue by moigagoo** [nim-lang/community_map#2]
How do I add my location to the file? Is it something like that?
```
{
  {
    "name": "Constantine Molchanov",
    "email": "moigagoo@live.com",
    "location": "Ulyanovsk, RU"
  },
}
```

**Comment by dom96** [comment]
Indeed, can we revive this? It's still on our community page (https://nim-lang.org/community.html), would be nice to get it going.

**Comment by moigagoo** [comment]
I'm all for it. I think, we should ask people to add their locations through the blog. Also, does this repo even get caught up by the website deployment routine?

**Comment by jabbalaci** [comment]
It's empty. Does it mean there are no Nim users? Either remove it or revive it.
